Description: Convert TeX hyphenation patterns to LibreOffice syntax
 This patch removes the TeX comments and patterns boundaries,
 and applies an encoding header for LibreOffice parsing.
Author: Ilyas Gasanov <torso.nafi@gmail.com>
Last-Update: 2013-05-16
Index: hyphen-ru/ruhyphal.tex
===================================================================
--- hyphen-ru.orig/ruhyphal.tex	2003-03-10 16:28:20.000000000 +0300
+++ hyphen-ru/ruhyphal.tex	2013-05-13 11:52:42.035372253 +0400
@@ -1,19 +1,4 @@
-% ruhyphal.tex:  Russian hyphenation patterns, version 2003/03/10
-% Copyright 1999-2003 Alexander I. Lebedev <swan@scon155.phys.msu.su>
-%
-% This program may be distributed and/or modified under the conditions
-% of the LaTeX Project Public License, either version 1.2 or any later
-% version.
-%
-% Patterns were generated with patgen from a 990,000-word list and then
-% manually corrected.
-%
-% The program consists of the files ruhyphal.tex, cyryoal.tex and two
-% document files README.ruhyphal and hyphen.rules.  The file cyryoal.tex
-% can be regenerated using mkcyryo script (a part of ruhyphen package)
-% and the latest release of rus-ispell dictionaries
-% <ftp://scon155.phys.msu.su/pub/russian/ispell/>.
-\patterns{
+KOI8-R
 .аб1р
 .аг1ро
 .ади2
@@ -4692,4 +4677,3 @@
 юсом
 ядо-зу-бе
 ярем-но-го
-}
